lib/facilities/metadata/configuration.rb

```diff
@@ -0,0 +1,35 @@
+# frozen_string_literal: true
+
+require 'common/client/configuration/rest'
+require 'common/client/middleware/response/raise_error'
+require 'common/client/middleware/response/facility_parser'
```

Comment:
I realize you just copy/pasted these lines, but can these be removed?
I'm wondering if they are relics from a time when autoloading wasn't configured properly for `lib`.
`facility_parser` isn't even used in this file.
I commented them out (and the `require` in the `client.rb`), and all specs pass.  But I'm worried I'm missing something.

Reply:
Ok, awesome. I'll take a look too!